doc/programming_guide/src/vanilla_api.rst

```diff
@@ -162,7 +162,7 @@ listed in your application's XN file (e.g. ``PORT_MCLK_IN_OUT``). For example:
 
 ``MIC_ARRAY_CONFIG_PORT_MCLK``
   Identifier of the 1-bit port on which the device is receiving the master audio
-  clock. Defaults to ``PORT_MCK_IN_OUT``.
+  clock. Defaults to ``PORT_MLCK_IN_OUT``.
```

Comment:
Good catch of the previous mis-spelling.  However, I think you mean `PORT_MCLK_IN_OUT`.